src/rgw/store/sfs/sqlite/objects/object_definitions.h

```diff
@@ -47,4 +47,20 @@ struct DBOPObjectInfo {
   RGWAccessControlPolicy acls;
 };
 
+struct DBMultipartObject {
+  std::string object_id;
+  std::string bucket_name;
+  std::string name;
+  std::string upload_id;
+  int state;  
+};
+
+struct DBOPMultipartObject {
+  uuid_d uuid;
+  std::string bucket_name;
+  std::string name;
+  std::string upload_id;
+  int state;
+};
+
```

Comment:
I think this is not used, at least in this actual PR.
I guess it's here for making parts persist over crashes in the future?

Reply:
it was forgotten there. but yes, it's what I'm looking into now. I'll drop it from the PR.